server/public/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Do we need this? Normally SPA is built to `server/public` folder, so it will provide it's own index.html. I don't know how we can make this build automatic, for now there is a bash script in `client` folder that will do it.

Same goes for `favicon.ico` you've added above - it will be provided upon SPA build.

I don't know what would be the best way to organize code so that SPA is served by server side. Usually I was doing SPA apps that were using separate ports, but then one cannot use cookies for authentication etc. and I wanted to keep it simple.